client/scripts/controllers/server/chain_entities.ts

```diff
@@ -23,6 +23,15 @@ export enum EntityRefreshOption {
   Nothing = 'nothing',
 }
 
+export function chainToEventNetwork(c: ChainInfo): SupportedNetwork {
+  if (c.base === 'substrate') return SupportedNetwork.Substrate;
+  if (c.network === ChainNetwork.ERC20) return SupportedNetwork.ERC20;
+  if (c.network === ChainNetwork.Compound) return SupportedNetwork.Compound;
+  if (c.network === ChainNetwork.Aave) return SupportedNetwork.Aave;
+  if (c.network === ChainNetwork.Moloch) return SupportedNetwork.Moloch;
+  throw new Error(`Invalid event chain: ${c.id}, on network ${c.network}, base ${c.base}`);
+}
```

Comment:
So we'll still maintain this list in the client of supported networks in relation to chain-events?

Reply:
Replying to the above:
> Haven't QA'd the client yet, but the code changes look 🔥 Great job on this massive cleanup

This is not directly relevant to this specific PR, but what is the/your QA process? is it possible to set up integration / UI tests with a CI service?

Reply:
We used to have a selenium test suite years ago but I'm not sure what happened to it. Should be easy to add via github events if we wanted to set something up again.